lib/net/dns/forward_record.rb

```diff
@@ -0,0 +1,40 @@
+module Net
+  module DNS
+    class ForwardRecord < DNS::Record
+      def to_s
+        "#{hostname}/#{ip}"
+      end
+
+      def destroy
+        super
+        proxy.delete(hostname)
+      end
+
+      def create
+        super
+        proxy.set attrs
+      rescue RestClient::Conflict
+        raise generate_conflict_error
+      end
+
+      # Returns an array of record objects which are conflicting with our own
+      def conflicts
+        @conflicts ||= [dns_lookup(hostname, :ipversion => self.ipversion)].delete_if { |c| c == self }.compact
```

Comment:
DNS orchestration doesn't work because `dns_lookup` calls in this file use 2 arguments. `ipversion` doesn't need to be passed